### ISSUE TITLE
Handle complexities for footer/contentinfo landmark

### ISSUE THREAD

**Issue by MelSumner** [ember-a11y/ember-a11y-landmarks#6]
The HTML5 `footer` element is not considered a `contentinfo` landmark when it is descendant of any of following elements:

- article
- aside
- main
- nav
- section

- [ ] To make this better for Ember applications, we should ensure that the `contentinfo` landmark is used appropriately. There may be some conversation/thought needed around how opinionated this addon wants to be regarding footer HTML element use.

**Comment by jenweber** [comment]
It seems that there are similar rules for banners https://www.w3.org/TR/wai-aria-practices/examples/landmarks/banner.html

**Comment by MelSumner** [comment]
@jenweber yes, you're correct. I might have it do some checking about what the parent element is and leave it at that.
